src/javascript-wrapper.js

```diff
@@ -64,26 +36,19 @@ var fRunUpdater = function(updater, updaterFormals, vni) {
      }).then(function(results) { 
          // console.log('updater returned results: ',results);
 
-         if (! _.isUndefined(results)) { 
+         if (! _.isUndefined(results)) {
              // Got some results back from updater
              // If the updater returned anything, set the output state with it
-             var newState = vni.outputState();
-             if (newState.lm === oldOutputStateLm || 
-                 _.isUndefined(oldOutputStateLm) && ! _.isUndefined(newState.lm)) { 
+             var newStateLm = vni.outputState().lm;
+             if (newStateLm === oldOutputStateLm ||
+                 _.isUndefined(oldOutputStateLm) && ! _.isUndefined(newStateLm)) {
```

Comment:
Yes, the data should get passed downstream.  But won't that happen if newStateLm is used as the value?   I would think that framework-ondata would notice that it differs from the old lm (which is undefined), so it would trigger an event to be sent downstream.  It may not matter much, but I was thinking that if the user went to the trouble of explicitly setting the lm in the updater, then maybe the user has a reason.